persistence/persistence-mybatis/src/test/java/org/cbioportal/persistence/mybatis/AlterationMyBatisRepositoryTest.java

```diff
@@ -59,7 +59,7 @@ public void setup() {
         sampleIdToProfileId.add(new MolecularProfileCaseIdentifier("TCGA-A1-A0SH-01", "study_tcga_pub_mutations"));
         sampleIdToProfileId.add(new MolecularProfileCaseIdentifier("TCGA-A1-A0SI-01", "study_tcga_pub_mutations"));
         sampleIdToProfileId.add(new MolecularProfileCaseIdentifier("TCGA-A1-A0SO-01", "study_tcga_pub_mutations"));
-        sampleIdToProfileId.add(new MolecularProfileCaseIdentifier("TCGA-A1-A0SP-01", "study_tcga_pub_mutations"));
+        sampleIdToProfileId.add(new MolecularProfileCaseIdentifier("TCGA-A1-A0SP-01", "study_tcga_pub_gistic"));
```

Comment:
1) Can you elaborate on why this change was introduced? I think that what you do here is include a request for counts for a sample _TCGA_A1_A0SP_01_ in profile _study_tcga_pub_gistic_ in which this sample does not have any observations. Is my conclusion correct?

2) You updated an existing test as a regression test for the problem fixed in this PR. For this you updated, rather than added a test. I propose to add a separate test in which you do `sampleToProfileId.put(new MolecularProfileCaseIdentifier("TCGA-A1-A0SP-01", "study_tcga_pub_gistic"), 3)` and in which you explicitly refer to the behavior under test (e.g. _testNoCountsIncludedWhenNoProfileMatch_ or something of the like).

Reply:
Yes, updated